service/history/workflow/update/abort_reason.go

```diff
@@ -45,8 +44,6 @@ func (r AbortReason) Error() error {
 		return registryClearedErr
 	case AbortReasonWorkflowCompleted:
 		return consts.ErrWorkflowCompleted
```

Comment:
Probably should be renamed to ErrWorkflowClosed

Reply:
Agree, I will do in separate PR.